modules/db/src/main/scala/Db.scala

```diff
@@ -238,6 +241,19 @@ private object Sql:
     val titles = title.values.list(n)
     sql"(p.title IN ($titles) OR p.women_title in ($titles))"
 
+  private def hasTitle: Boolean => AppliedFragment =
+    case true  => sql"p.title IS NOT NULL" apply (Void)
+    case false => sql"p.title IS NULL".apply(Void)
```

Comment:
Style inconsistency :(

Reply:
yeah, I think compiler should warn about this because `apply` is not an `infix` operator. Maybe the sbt-tpolecat plugin does something funny here 🤔

Reply:
I fixed both comments and force pushed. Thanks!